### ISSUE TITLE
Sometimes app crashes in TSMarkdownParser init

### ISSUE THREAD

**Issue by YuriSolodkin** [laptobbe/TSMarkdownParser#58]
Sometimes app crashes in TSMarkdownParser init:

```
Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x2584985b __exceptionPreprocess
1  libobjc.A.dylib                0x37422dff objc_exception_throw
2  CoreFoundation                 0x2576734b -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]
3  CoreFoundation                 0x257671cf +[NSDictionary dictionaryWithObjects:forKeys:count:]
4  Service                        0x1dd60f -[TSMarkdownParser init] (TSMarkdownParser.m:35)
5  Service                        0x1dd921 +[TSMarkdownParser standardParser] (TSMarkdownParser.m:55)

```

From the code I can see that it is possible that it crashes in:

```
_quoteAttributes = @[@{NSFontAttributeName: [UIFont fontWithName:@"HelveticaNeue-Italic" size:defaultSize]}];
```

or

```
    _monospaceAttributes = @{ NSFontAttributeName: [UIFont fontWithName:@"Courier New" size:defaultSize],
                              NSForegroundColorAttributeName: [UIColor colorWithRed:0.95 green:0.54 blue:0.55 alpha:1] };

```

On stackoverflow there is a discussion re HelveticaNeue-Italic.
http://stackoverflow.com/questions/19527962/what-happened-to-helveticaneue-italic-on-ios-7-0-3

I can see these crashes on iOS 9.1.0 and even 9.3.1, but cannot replicate on my iPads or emulator
Maybe it will be safer to add something like:

```
    UIFont *quoteFont = [UIFont fontWithName:@"HelveticaNeue-Italic" size:defaultSize];
    if (quoteFont == nil) {
        quoteFont = [UIFont italicSystemFontOfSize:defaultSize];
    }
    if (quoteFont == nil) {
        quoteFont = [UIFont systemFontOfSize:defaultSize];
    }
    _quoteAttributes = @[@{NSFontAttributeName: quoteFont}];

```

**Comment by Coeur** [comment]
I'll accept a pull request, so you get credit for the fix. :)

**Comment by YuriSolodkin** [comment]
Ok, thanks, I created a pull request.

**Comment by afocsa** [comment]
Hi @Coeur 
Any chances to merge the PR to fix this issue?
Thanks.

**Comment by Coeur** [comment]
@afocsa , oh, thank you for reminding me... I had actually forgot about it, I'm really sorry.
This weekend!

**Comment by Coeur** [comment]
Fixed in 2.1.4.
